### ISSUE TITLE
Workers need to be able to create new resque jobs via a REST API on baw-server

### ISSUE THREAD

**Issue by cofiem** [QutBioacoustics/baw-workers#7]
See https://github.com/QutBioacoustics/baw-server/issues/193

This includes ensuring there is a generic way to log in to the baw-server website.

This will mainly be used for harvesting - once harvesting is complete, queue jobs for processing the audio to create visualisations.

**Comment by atruskie** [comment]
For reference, harvester should also queue the 'copy to availae' job as well - is there an issue tracking that requirement?

**Comment by atruskie** [comment]
#11 tracks the copy task

**Comment by cofiem** [comment]
I'm not sure this is still relevant, as can now can enqueue resque tasks from other resque tasks.

**Comment by atruskie** [comment]
Fair enough

**Comment by cofiem** [comment]
See #21

**Comment by cofiem** [comment]
Closing in favour of #21
